pyproject.toml

```diff
@@ -267,6 +267,7 @@ filterwarnings = [
     # Unexpected warnings, worth investigating
     # Lightning is having trouble inferring the batch size for ChesapeakeCVPRDataModule and CycloneDataModule for some reason
     "ignore:Trying to infer the `batch_size` from an ambiguous collection:UserWarning",
+    "ignore: Tight layout not applied. The bottom and top margins cannot be made large enough:UserWarning",
```

Comment:
What causes this error and why should we ignore it?

Reply:
This was originally caused by calling `plt.tight_layout`  but I was able to resolve it now by setting the `constrained` flag in the subplot function.

Reply:
Ok, so setting layout as a parameter is only possible for `matplotlib>=3.5.0`. This means that we either bump matplotlib's min version or ignore this warning.

Reply:
Can we instead change the size of the figure before calling tight_layout? Would like to avoid bumping matplotlib version if we don't need to.